test/Microsoft.Azure.WebJobs.Host.FunctionalTests/ILoggerTests.cs

```diff
@@ -198,6 +232,18 @@ public void ILogger(ILogger log)
                 log.LogError(0, ex, "Log {other} keys {and} values", "3", "4");
             }
 
+            [NoAutomaticTrigger]
+            [FixedDelayRetry(3, "00:00:01")]
```

Comment:
Since the actual duration doesn't matter, can you use "00:00:00.500" or something so it doesn't run for 3 seconds unecessarily?